src/cmd/add.nim

```diff
@@ -5,13 +5,32 @@ import ../cli/term, ../cli/prompt
 import ../util/flow
 import ../modpack/files, ../modpack/install
 
+proc addDependencies(manifest: var Manifest, file: ManifestFile,
+    strategy: string): void =
+  ## Recursively add dependencies of a mod
+  if len(file.dependencies) > 0:
```

Comment:
You can remove the `if` here, because if `file.dependencies` is empty, the `for id in file.dependencies` will just do.. nothing ;)